### DISCUSSION TITLE
php version 7.3

### DISCUSSION THREAD

**Discussion by GNIKEI-D** [spatie/laravel-short-schedule#53] [Q&A]
hi, can i run this on php v7.3?
im getting this error when running php artisan short-schedule:run

  syntax error, unexpected 'LoopInterface' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)

**Comment by freekmurze** [comment]
No, see: https://github.com/spatie/laravel-short-schedule/blob/6fc08409f8b7205dd9ccc5bf5c7196f72372f1b0/composer.json#L19

**Reply by GNIKEI-D** [reply]
not even old versions will work? on 7.4?
